README.md

```diff
@@ -1,289 +1,613 @@
+
+A Synthetic Provisioned Service resource **MUST** define a `.spec.services` which is an array of `ObjectReference`-ables to a resource.
 
+A Synthetic Provisioned Service resource **MUST** define a `.spec.mappings` which is an array of `Mapping` objects. A `Mapping` object **MUST** define `name` and `value` entries. The value of a `Mapping` **MAY** contain zero or more tokens beginning with `((`, ending with `))`, and encapsulating a [JSON Path](https://kubernetes.io/docs/reference/kubectl/jsonpath/) to an entry on a resource defined in `services`. The value of this `Secret` entry **MUST** be substituted into the original value string, replacing the token. Once all tokens have been substituted, the new value **MUST** be added to the binding `Secret` exposed by the resource.
 
-#### Exposing data as environment variables
+If a `.spec.type` is set, the `type` entry in the binding `Secret` **MUST** be set to its value. If a `.spec.provider` is set, the `provider` entry in the binding `Secret` **MUST** be set to its value.
 
-The specification allows for binding properties to be additionally exposed as environment variables.  The assumption is that applications will have pre-knowledge of these environment variable keys, so the mechanisms described in this section are solely for the purpose of allowing the entity responsible for mounting the binding data to know which keys should also be made available as environment variables.  
+A Synthetic Provisioned Service resource **MUST** define a `.status.binding.name` which is a `LocalObjectReference`-able to a `Secret`. The `Secret` **MUST** be in the same namespace as the resource. The `Secret` **MUST** contain a `type` entry with a value that identifies the abstract classification of the binding. It is **RECOMMENDED** that the `Secret` also contain a `provider` entry with a value that identifies the provider of the binding. The `Secret` **MAY** contain any other entry.
 
-This decision is made in the following ascending order of precedence:
-* value of `ServiceBinding`'s global `bindAs` element, which applies to all binding data.
-* value of the `bindAs` attribute in each of the `dataMappings` elements inside `ServiceBinding`.
+### Resource Type Schema
+
+```yaml
+apiVersion: service.binding/v1alpha1
+kind: SyntheticProvisionedService
+metadata:
+  name:         # string
+spec:
+  type:         # string, optional
+  provider:     # string, optional
+
+  services:     # []ObjectReference-able
+  - apiVersion: # string
+    kind:       # string
+    name:       # string
+    ...
+
+  mappings:     # []Mapping, optional
+  - name:       # string
+    value:      # string
+
+status:         # Provisioned Service Duck Type
+  binding:
+    name:       # string
+```
+
+### Example Resource
+
+```yaml
+apiVersion: service.binding/v1alpha1
+kind: SyntheticProvisionedService
```

Comment:
how about calling this `ComposedService`?

Reply:
Opened this issue https://github.com/application-stacks/service-binding-specification/issues/53 for it.